crates/jstzd/src/task/octez_client.rs

```diff
@@ -123,38 +123,49 @@ impl OctezClient {
         Ok(command)
     }
 
-    fn spawn_command<S: AsRef<OsStr>, I: IntoIterator<Item = S>>(
+    async fn spawn_and_wait_command<S: AsRef<OsStr>, I: IntoIterator<Item = S>>(
         &self,
         args: I,
-    ) -> Result<Child> {
+    ) -> Result<()> {
```

Comment:
I think this should return `Result<Output>`

Reply:
i don't think we need it now unless we have a method that needs the stdout. The stdout/stderr gets printed out in the cli and i think that's sufficient. do you have anything in mind that would require the stdout from the client?

Reply:
Yes, stdout will be parsed to get output for commands later. Making the change now means that your commands in this PR will not need to be refactored later.

Reply:
ok i i will add this for now and test it in the following PR for show address